sdk/servicebus/Azure.Messaging.ServiceBus/src/EntityNameFormatter.cs

```diff
@@ -128,5 +128,20 @@ private static void CheckValidEntityName(string entityName, int maxEntityNameLen
                 }
             }
         }
+
+        private static string GetPathWithoutBaseUri(string entityName)
+        {
+            // Note: on Linux/macOS, "/path" URLs are treated as valid absolute file URLs.
+            // To ensure relative queue paths are correctly rejected on these platforms,
+            // an additional check using IsWellFormedOriginalString() is made here.
+            // See https://github.com/dotnet/corefx/issues/22098 for more information.
+            if (Uri.TryCreate(entityName, UriKind.Absolute, out Uri uriValue) &&
+                uriValue.IsWellFormedOriginalString())
```

Comment:
Keep in mind that this check can be expensive. I looked into this before for the code generator. Instead, we opted to validate that the string starts with a certain criteria (in our case, we only needed to validate `http://`). Here, I don't know what format the URI needs to be. But, it is much cheaper to check the start of the string that to use the URI validation methods.

Reply:
That's fair. Perhaps we can check the scheme property. The point of the check is to disallow "/path" from being considered a valid Uri.

Reply:
I'm not sure about changing this for now. We can revisit this later.